lib/log.js

```diff
@@ -8,15 +8,20 @@ import { Tail } from 'tail'
 import { platform } from 'node:os'
 import { on } from 'node:events'
 
-export const formatLog = (text, date = new Date()) =>
-  text
+export const formatLog = (
+  text,
+  { date = new Date(), isHumanReadable = false } = {}
```

Comment:
I find the name `isHumanReadable` a bit misleading since it does not tell us whether the value _is_ human readable now but whether we want to produce a human-readable representation in the output.

How about using just `humanReadable: true`?

Alternatively, what do you think about using `pretty: true` instead?

Reply:
Went with `pretty`